templates/Layout/default.twig

```diff
@@ -1,28 +1,33 @@
 <!DOCTYPE html>
 <!--
 
-██████╗ ███████╗██████╗ ██╗████████╗ █████╗     ██╗  ██╗
-██╔══██╗██╔════╝██╔══██╗██║╚══██╔══╝██╔══██╗    ██║  ██║
-██████╔╝█████╗  ██║  ██║██║   ██║   ███████║    ███████║
-██╔══██╗██╔══╝  ██║  ██║██║   ██║   ██╔══██║    ╚════██║
-██████╔╝███████╗██████╔╝██║   ██║   ██║  ██║         ██║
-╚═════╝ ╚══════╝╚═════╝ ╚═╝   ╚═╝   ╚═╝  ╚═╝         ╚═╝
-
-BEdita 4 ~ @ 2018 from ChannelWeb & Chialab with love
+██████╗░███████╗██████╗░██╗████████╗░█████╗░  ███╗░░░███╗░█████╗░███╗░░██╗░█████╗░░██████╗░███████╗██████╗░
+██╔══██╗██╔════╝██╔══██╗██║╚══██╔══╝██╔══██╗  ████╗░████║██╔══██╗████╗░██║██╔══██╗██╔════╝░██╔════╝██╔══██╗
+██████╦╝█████╗░░██║░░██║██║░░░██║░░░███████║  ██╔████╔██║███████║██╔██╗██║███████║██║░░██╗░█████╗░░██████╔╝
+██╔══██╗██╔══╝░░██║░░██║██║░░░██║░░░██╔══██║  ██║╚██╔╝██║██╔══██║██║╚████║██╔══██║██║░░╚██╗██╔══╝░░██╔══██╗
+██████╦╝███████╗██████╔╝██║░░░██║░░░██║░░██║  ██║░╚═╝░██║██║░░██║██║░╚███║██║░░██║╚██████╔╝███████╗██║░░██║
+╚═════╝░╚══════╝╚═════╝░╚═╝░░░╚═╝░░░╚═╝░░╚═╝  ╚═╝░░░░░╚═╝╚═╝░░╚═╝╚═╝░░╚══╝╚═╝░░╚═╝░╚═════╝░╚══════╝╚═╝░░╚═╝
+
+BEdita Manager
 -->
 
 <html lang="{{ config('I18n.lang', 'en')}}">
 <head>
     {{ Html.charset()|raw }}
-    {{ element('meta') }}
-    <title>{{ "#{_view.fetch('title')} | #{project.name ?: 'BEdita 4'}" }}</title>
+
+    {{ Html.meta('viewport', 'width=device-width, initial-scale=1.0')|raw }}
+    {{ Html.meta('description', 'BEdita Manager')|raw }}
+    {{ Html.meta({
```

Comment:
we could skip this `meta rel=manifest` - I don't think it applies in our case https://developer.mozilla.org/en-US/docs/Web/Manifest